crates/bevy_audio/src/sinks.rs

```diff
@@ -15,6 +15,11 @@ pub trait AudioSinkPlayback {
     ///
     /// The value `1.0` is the "normal" volume (unfiltered input). Any value other than `1.0`
     /// will multiply each sample by this value.
+    ///
+    /// # Note on Audio Volume
```

Comment:
```suggestion
    /// # Note on Audio Volume
    ///
```
not a hard rule, but it's nicer to keep a new lines after heading